includes/size/ios.md

```diff
@@ -0,0 +1,2 @@
+!!!info "SDK bundle size"
+    You can get the SDK bundle size by using the [Cocoapods size measurement tool](https://github.com/google/cocoapods-size).
```

Comment:
Should we link to specific SDK?

Reply:
Updated it to 
![image](https://github.com/Amplitude-Developer-Docs/amplitude-dev-center/assets/31029607/c4be9e25-90e4-4b27-94ae-3d9a1da74d8c)